engine/src/main/java/org/terasology/rendering/AABBRenderer.java

```diff
@@ -102,8 +104,9 @@ public void renderSolid() {
         CoreRegistry.get(ShaderManager.class).enableDefault();
 
         glPushMatrix();
-        Vector3f cameraPosition = CoreRegistry.get(LocalPlayer.class).getViewPosition();
-        glTranslated(aabb.getCenter().x - cameraPosition.x, -cameraPosition.y, aabb.getCenter().z - cameraPosition.z);
+        Vector3f cameraPosition = CoreRegistry.get(LocalPlayer.class).getViewPosition(new Vector3f());
+        Vector3f center = aabb.center(new Vector3f());
+        glTranslated(center.x - cameraPosition.x, -cameraPosition.y, center.z - cameraPosition.z);
```

Comment:
I'd like to avoid this code duplication, but that's probably something for another day...